Assets/Scripts/Game/Leveling/AlignmentSystem.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.Events;
+
+namespace RPG
+{
+    public class AlignmentSystem : MonoBehaviour
+    {
+        public UnityEvent<int> onAligneChanged;
+
+        public int Sexiness
+        {
+            get => GameManager.Player.GetProperty<int>(PropertyName.Sexiness);
+            set => GameManager.Player.SetProperty<int>(PropertyName.Sexiness, value);
+        }
+
+        public int Morals
+        {
+            get => GameManager.Player.GetProperty<int>(PropertyName.Morals);
+            set => GameManager.Player.SetProperty<int>(PropertyName.Morals, value);
+        }
+
+        public int Leanings
+        {
+            get => GameManager.Player.GetProperty<int>(PropertyName.Leanings);
+            set => GameManager.Player.SetProperty<int>(PropertyName.Leanings, value);
+        }
+
+        void Awake()
+        {
+            GameManager.AddPlayerCreatedListener(OnPlayerCreated);
+            
+            onAligneChanged.AddListener(UpdateMorals);
+            onAligneChanged.AddListener(UpdateLeanings);
+            onAligneChanged.AddListener(UpdateSexiness);
```

Comment:
Generally, events should be invoked when the thing they're referencing changes, not the other way around. So, instead of having a listener that updates the individual alignment properties when alignment changes, invoke the "alignment changed" event any time any of the properties are updated. So in UpdateMorals you would say `OnAlignmentChanged.Invoke()`. And I guess we'd probably make it so that it returns the three floats when it's invoked, so for UpdateMorals for example you would actually say: `OnAlignmentChanged.Invoke(points, 0, 0)` or something similar. Let me know if you wanna discuss the events for this. :)